sdk/shared/sys/icc.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * icc.c
+ *
+ *  Created on: 7 nov. 2023
+ *      Author: pnowa
+ */
+
+#include "icc.h"
+
+///////////////////////////////////////////////////////
+// KEEP THIS FILE IN SYNC IN BOTH CPU0 AND CPU1
+///////////////////////////////////////////////////////

```

Comment:
Hm. Maybe this should say something else since this exact file is include for both cores?

Reply:
Yep, I noticed this as well and corrected it in my work today.